Dockerfile

```diff
@@ -36,9 +36,15 @@ RUN adduser --gecos '' --disabled-password coder && \
 
 USER coder
 # We create first instead of just using WORKDIR as when WORKDIR creates, the user is root.
-RUN mkdir -p /home/coder/project
+RUN mkdir -p /home/coder/project && \
+    chmod g+rw /home/coder/project;
```

Comment:
Why is this necessary?

Reply:
I need to set the R/W permissions explicitly because I found out on OpenShift hosts, if this is not set, the container would outright crash.

Reply:
Still not clear why this is necessary. You shouldn't be able to write if you don't own it, that is correct behaviour. I'm going to revert this @kylecarbs

Reply:
`container would outright crash` <-- still a little concerned about the possibility of reintroducing this possibility with a revert.

Reply:
> container would outright crash <-- still a little concerned about the possibility of reintroducing this possibility with a revert.
 
Not a concern, don't worry. Its definitely not outright crashing, there is a permission related issue.

Reply:
@Multishifties I explained it already becuase of how OpenShift handles UIDs and GIDs, which would make `/home/coder/project` inaccessible practically on the official image. So I guess this would make sense if they revert it, but the official image will never work on OpenShift due to missing permissions

Reply:
> becuase of how OpenShift handles UIDs and GIDs, 

What does that mean? How does it handle it?

You're not being clear with your concerns, I can't give you a good response.